src/current/_includes/cockroachcloud/download-the-binary.md

```diff
@@ -1,21 +1,29 @@
     <section class="filter-content" markdown="1" data-scope="mac">
-    If you have not done so already, run the first command in the dialog to install the CockroachDB binary in `/usr/local/bin`, which is usually in the system `PATH`. To install it into a different location, replace `/usr/local/bin/`.
+    If you have not done so already, run the first command in the dialog to install the latest downloadable version of the CockroachDB binary in `/usr/local/bin`, which is usually in the system `PATH`. To install it into a different location, replace `/usr/local/bin/`.
 
     {% include_cached copy-clipboard.html %}
     ~~~ shell
-    curl https://binaries.cockroachdb.com/cockroach-{{ page.release_info.version }}.darwin-10.9-amd64.tgz | tar -xJ && sudo cp -i cockroach-{{ page.release_info.version }}.darwin-10.9-amd64/cockroach /usr/local/bin/
+    curl https://get.cockroachdb.com/dl/cockroach-latest.darwin-10.9-amd64.tgz | tar -xJ && sudo cp -i cockroach-latest.darwin-10.9-amd64/cockroach /usr/local/bin/
     ~~~
 
+    {{site.data.alerts.callout_success}}
+    To download the ARM 64-bit architecture instead of the Intel 64-bit binary, replace all instances of `darwin-10.9-amd64` with `darwin-11.0.arm64` before running the command.
```

Comment:
this doesn't work(?)

Works (you can try this from a browser):
https://get.cockroachdb.com/dl/cockroach-latest.darwin-10.9-amd64.tgz

Doesn't work (I get a Not found):
https://get.cockroachdb.com/dl/cockroach-latest.darwin-11.0.arm64.tgz

(let's leave this out then?)

Reply:
How come this doesn't work for the ARM binaries? What are ARM clients supposed to do to get the latest binary?

Reply:
🤦🏼 I added a dot instead of a dash. 

this works:
https://get.cockroachdb.com/dl/cockroach-latest.darwin-11.0-arm64.tgz

Reply:
> `darwin-10.9-amd64` with `darwin-11.0.arm64`

oh so then instead of `darwin-11.0.arm64` it should be `darwin-11.0-arm64`

Reply:
Now that I am not exposing the curl commands, this is not an issue (I suppose these will be exposed in the modal when RE-565 is addressed. Thanks for confirming that the ARM downloads do download in the browser.

Reply:
The URL is incorrect, need to replace one of the dots to a dash:
https://get.cockroachdb.com/dl/cockroach-latest.darwin-11.0-arm64.tgz